http_server/logout.php

```diff
@@ -24,7 +24,9 @@
     // client has token, token sent via post
     if (!is_empty($_POST['token']) && is_empty($data)) {
         $ret->errorType = 'token';
-        token_delete($pdo, $token);
+        if (token_delete($pdo, $token, true) === false) {
```

Comment:
The exploit being fixed: https://gitlab.com/TheMightyBacon/pr2logoutexploitbypass1